lms/djangoapps/ora_staff_grader/tests/test_serializers.py

```diff
@@ -87,8 +92,21 @@ def test_team_ora(self):
             "name": self.ora_data['display_name'],
             "prompts": self.ora_data['prompts'],
             "type": "team",
+            "textResponseConfig": "none",
+            "fileUploadResponseConfig": "none",
         }
 
+    @ddt.unpack
+    @ddt.data((None, None), ('optional', 'optional'), ('required', 'required'))
+    def test_response_config(self, text_response, file_upload_response):
+        self.mock_ora_instance.text_response = text_response
+        self.mock_ora_instance.file_upload_response = file_upload_response
+
+        data = OpenResponseMetadataSerializer(self.mock_ora_instance).data
+
+        assert data['textResponseConfig'] == text_response or "none"
+        assert data['fileUploadResponseConfig'] == file_upload_response or "none"
```

Comment:
Just a nit: it's not usually good practice to duplicate code directly between implementation and test. This would be less fragile as a test for 'optional'/'required' and a second test for the `None` case.